server/tests/conftest.py

```diff
@@ -352,8 +351,26 @@ async def with_preloading(sdb, mdb, mdb_rw, pdb, rdb, with_preloading_enabled):
 
 @pytest.fixture(scope="function")
 async def app(metadata_db, state_db, precomputed_db, persistentdata_db, slack,
-              with_preloading_enabled) -> AthenianApp:
+              with_preloading_enabled, request) -> AthenianApp:
+    """Build the especifico App to be used during tests
+
+    By default handler responses will be validated against oas spec,
+    `app_validate_response` mark can be applied to disable this behavior:
+
+    ```
+    @pymark.mark.app_validate_response(False)
```

Comment:
What about using the already provided `pytest.mark.xfail` mark with a custom reason? Or a wrapper around that. This would have the advantage that it supports some handy helpers such as passing `strict` arg or totally ignoring the marks with `pytest --runxfail`.

Reply:
You could eventually use `pytest.mark.xfail` with the `condition` that comes from an env var something like `ENABLE_RESPONSE_SCHEMA_CHECK=1`.

Reply:
The problem is if the test fails because we screwed the logic we won't detect it anymore with xfail.

Reply:
typo fixed; yes using xfail will make those tests basically useless in the CI test execution